public/src/actions/calls.jsx

```diff
@@ -78,6 +78,7 @@ export function assignToCall(userId, campaignId) {
     .then((call) => {
       const { data: callObj } = call;
       const { status, outcome } = callObj;
+      console.log('SHOULD BE NEW CALL: ', callObj.id, status);
```

Comment:
Can we drop this if it was for debugging?

Reply:
lol sorry